### PR TITLE
adding "transaction" and "multiplier"

### PR DESCRIPTION
**Pull request by philipeachille**

### CODE REVIEW

**Comment by T1xx1** [comment]
@kisvegabor can you merge this branch?

**Review by kisvegabor — COMMENTED** [review]
Sorry, I forgot to actually send my review... well... in October. 
Shame on me.

File: README.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ Of course, you can consider using the not recommended ones. For example why not
 -  **mng** manager
 -  **mod** modulo
 -  **msg** message
+-  **mplr** multiplier
```
[the remark this answers is not in the document]
Comment (philipeachille) [reply README.md:203]: no, I can't show "mplr" anywhere, it is one i made up in one of my projects :)

[the remark this answers is not in the document]
Comment (T1xx1) [reply README.md:203]: What about "mltpr"

**Pull request closed without merging**